code/modules/atmospherics/gasmixtures/gas_types.dm

```diff
@@ -1,4 +1,4 @@
-var/list/hardcoded_gases = list("o2","n2","co2","plasma") //the main four gases, which were at one time hardcoded
+var/list/hardcoded_gases = list("o2","n2","co2","plasma", "freon") //the main four gases, which were at one time hardcoded
```

Comment:
No. I already told robustin no on this.
you want it in the scanner, make the scanner use a new list.